extraeditor.sh

```diff
@@ -0,0 +1,18 @@
+#! /bin/bash
+
+currdirectory=$(pwd)
+newfolder=$currdirectory/../Parking3
+
+mkdir -p $newfolder
+
+# Project files
+cp -a $currdirectory/ProjectFiles $newfolder/
+
+rm -rf $newfolder/ProjectFiles/Assets $newfolder/ProjectFiles/ProjectSettings
+
+ln -s $currdirectory/ProjectFiles/Assets $newfolder/ProjectFiles/Assets
+ln -s $currdirectory/ProjectFiles/ProjectSettings $newfolder/ProjectFiles/ProjectSetings
```

Comment:
Typo?

```suggestion
ln -s $currdirectory/ProjectFiles/ProjectSettings $newfolder/ProjectFiles/ProjectSettings
```

Reply:
Oh my fucking god that’s why it wasn’t working 😤